docs/testing.md

```diff
@@ -40,6 +40,10 @@ To run just a subset of them, specify a particular file or directory:
     py.test tests/unit
     py.test tests/unit/api.py
 
+To run just a pep8 test:
+
+    py.test --pep8 -m pep8
```

Comment:
In order to prevent running pep8 against included libraries, should this be:
```
py.test --pep8 -m pep8 haas/ tests/
```
?